### ISSUE TITLE
DateTime arithmetic doesn't factor in timezones.

### ISSUE THREAD

**Issue by sichbo** [bridgedotnet/Bridge#3437]
### Steps To Reproduce

```csharp
public class Program
{
    public static void Main()
    {
       Console.WriteLine(DateTime.Now - DateTime.UtcNow);
    }
}
```

### Expected Result

```
-04:00:00 (or whatever your timezone is)
```

### Actual Result

```
00:00:00
```

## See Also

- #3441

**Comment by geoffreymcgill** [comment]
Hi @sichbo – Thanks the report. It's unfortunate this defect was not discovered earlier. I think I see the problem and it's an easy fix. We will fix right away and include the fix in the next release.

**Comment by sichbo** [comment]
Thanks Geoff! Keep up the great work.

**Comment by geoffreymcgill** [comment]
A pull-request has been submitted with a fix for this issue. Once the pr is reviewed and any final testing is complete, the pr will be added to the next release. We are anticipating this fix will make it into the upcoming 16.8.2 release. 

Hope this helps.

Thanks again for reporting the defect.

**Comment by fabriciomurta** [comment]
The change looks good. Ensured there will be unit tests (well, that works as long as you are not in GMT/UTC, of course) so that this issue never comes back.

**Comment by geoffreymcgill** [comment]
Fixed and included in the Bridge 16.8.2 release.
